CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.2.0 - 2015-05-23
```

Comment:
This should be 2.0 since you are introducing a breaking change for pre 4.1 users. Am I right ?